### ISSUE TITLE
if the twitter native app is not installed not work

### ISSUE THREAD

**Issue by lianesten** [ManifestWebDesign/twitter-connect-plugin#45]
When the twitter native app is installed on device, the plugin work fine. But when is not insatlled not work. What is the best way to integrate browser login when the native twitter app is not installed?.
I am using Ionic 2.
I am going to implement [this twitter documentation](https://dev.twitter.com/web/sign-in/implementing). Let me know if there are betters ways to do it!

**Comment by ferdil** [comment]
That is correct, and it is by design. The plugin uses the native Frameworks to work.

**Comment by alstonkim** [comment]
@lianesten  could you please explain to me how you've handled this?
